pkgs/development/python-modules/pkg-about/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, importlib-metadata
+, importlib-resources
+, setuptools
+, packaging
+, tomli
+}:
+
+buildPythonPackage rec {
+  pname = "pkg-about";
+  version = "1.0.8";
+  disabled = pythonOlder "3.7";
+
+  format = "pyproject";
+  src = fetchPypi {
```

Easier for maintainer if the ordering matches the majority. Not a blocker, though.

```suggestion
  version = "1.0.8";
  format = "pyproject";

  disabled = pythonOlder "3.7";

  src = fetchPypi {
```